test/data/index.json

```diff
@@ -215,6 +215,12 @@
 		"parent": "extensions"
 	},
 	{
+		"name": "CompoundTree.html",
+		"url": "extensions/CompoundTree.html",
+		"title": "Test Grid Compound Columns with Tree",
+		"parent": "extensions"
+	},
+	{
```

Comment:
FYI, you can regenerate this automatically by running `node test/data/createIndex.js`.  It will pick up the title from the `title` tag in the file automatically.